src/main/resources/static/css/style.css

```
@@ -210,3 +210,77 @@ nav {
 .movie-container:hover .nav-btn {
     opacity: 1;
 }
+
+/* Login sida relaterad*/
```

Stylelint (17.7.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)